src/ASM/ASMbase.h

```diff
@@ -874,8 +893,9 @@ class ASMbase
   static int gNod; //!< Global node counter
 
 private:
-  std::pair<size_t,size_t> myLMs; //!< Nodal range of the Lagrange multipliers
-  std::vector<char>    myLMTypes; //!< Type of Lagrange multiplier ('L' or 'G')
+  std::vector<char> myLMTypes; //!< Type of Lagrange multiplier ('L' or 'G')
+  std::set<size_t>  myLMs;     //!< Nodal indices of the Lagrange multipliers
```

Comment:
are you sure we are not order dependent anywhere?

Reply:
Not sure what you mean